nautobot/dcim/templates/dcim/location.html

```diff
@@ -81,7 +81,7 @@
                 <td>Children</td>
                 <td>
                     {% if object.location_type.children.all %}
-                        <a href="{% url 'dcim:location_list' %}?parent={{ object.slug }}">{{ children_table.rows|length }}</a>
+                        <a href="{% url 'dcim:location_list' %}?parent={{ object.name }}">{{ children_table.rows|length }}</a>
```

Comment:
This should probably be `object.pk` since `name` is not unique on Location? Same comment throughout this file